index.html

```diff
@@ -133,12 +133,12 @@ <h1 id="text07" class="style1">The Cosmos Ladies</h1>
                                 <h3 id="text28" class="style3">Mission</h3>
                                 <p id="text29" class="style5">
                                     This platform was founded with the aim of simplifying technical concepts to make the
-                                    industry more accessible to under-represented people breaking into the
+                                    industry more accessible to under-represented people who are breaking into the
                                     industry.<br><br>
-                                    ⭐️ Onboard, Educate, & Support Web3 Developers <br>
-                                    ⭐️ Make Cosmos more visible among Women in Web3 <br>
+                                    ⭐️ Onboard, educate, and support Web3 developers <br>
+                                    ⭐️ Make Cosmos more visible among women in Web3 <br>
                                     ⭐️ Share knowledge about different projects in Cosmos ecosystem <br>
-                                    ⭐️ Inclusive and empowering community supporting women-led projects in the
+                                    ⭐️ Practice inclusive and empowering community practices to support women-led projects in the

```

Comment:
for parallel sentence construction, let's start each list item with a root verb "Practice" to go along with Onboard... , Make, and Share